packages/theme/src/themes/Bruttal/BruttalTheme.ts

```diff
@@ -573,4 +574,35 @@ export const BruttalTheme: Theme = {
       letterSpacing: 'normal',
     },
   },
+  reactAuth: {
```

Comment:
I don't feel safe having these variants now, and it's best to style the component. For example, in what case we'd use 

```
  container: {
    flexDirection: 'row', // row instead of column?
  },
``` 

?

Reply:
That make sense. As we talked in the Discord, I'll remove the variants for now and implement those changes in each component